hieradata/clients/vpn.jenkins.io.yaml

```diff
@@ -0,0 +1,11 @@
+--
+networks:
+  eth0: 
+    route-metric: 100
+    macaddress: 00:0d:3a:0e:4b:1c
+  eth1:
+    route-metric: 200
+    macaddress: 00:0d:3a:0e:4f:89
+  eth2:
+    route-metric: 300
+    macaddress: 00:0d:3a:0e:4e:7e
```

Comment:
There should be an EOL character at the end of this line.

Reply:
Done